miner/build.rs

```diff
@@ -0,0 +1,12 @@
+extern crate cc;
+
+fn main() {
+    cc::Build::new()
+        .file("src/worker/cuckoo.c")
+        .include("src/worker")
+        .flag("-O3")
+        .flag("-lcrypto")

```

Comment:
1. Have you tested in Windows?
2. Need to update docs about this new dependency.

Reply:
It's OK to drop Windows support for several versions, since this is not the final PoW function we will use.